doc/lua_api.txt

```diff
@@ -1936,6 +1936,11 @@ Call these functions only at load time!
 * `minetest.notify_authentication_modified(name)`
     * Should be called by the authentication handler if privileges changes.
     * To report everybody, set `name=nil`.
+* `minetest.check_password_entry(name, entry, password)`
+    * Returns true if the db entry for a player with name matches given
+    * password, false if not, nil if its formatting is invalid.
+    * Only use it for making it possible to log in via the password from
+    * via protocols like IRC, other uses (like inside the game) are frowned upon.
```

Comment:
Why not something like "stored authentication credentials"?

Just the word 'entry' alone is sort of ambiguous.  I certainly didn't know what this function was talking about until I actually looked at what the various functions called inside the API do.

Just because you're familiar with auth doesn't mean all the modders are too...  FWIW I was guilty of this in the past with mapgen related APIs.